mmv1/templates/terraform/resource.go.tmpl

```diff
@@ -675,11 +675,19 @@ func resource{{ $.ResourceName -}}Read(d *schema.ResourceData, meta interface{})
         }
     }
 {{-    else -}}
+{{- if $prop.WriteOnlyPairingExists -}}
+    if {{ $prop.Name -}}Wo, _ := d.GetRawConfigAt(cty.GetAttrPath("{{ underscore $prop.Name -}}_wo")); {{ $prop.Name -}}Wo.IsNull() {
```

Comment:
In my testing with write-only fields, we don't have access to the raw config during Read during refresh - only when Read is called as part of Create or Update. I think that would mean that the Wo property will always appear null during refresh, leading the field to still be (incorrectly) set.

I'd generally expect that fields with a write-only equivalent would be ignore_read and therefore wouldn't show up in ReadProperties at all - is that not the case?